rethinkdb_mock/db.py

```diff
@@ -3,99 +3,80 @@
 
 from future.utils import iteritems
 import rethinkdb
+import rethinkdb.ast as r_ast
 
 from . import rtime
 from . import util
-from .ast_base import BinExp
 from .rql_rewrite import rewrite_query
+from .ast_base import BinExp
 from .rql_rewrite import RQL_TYPE_TRANSLATIONS
```

Comment:
I don't think those two imports are required now. In fact, I think that I deleted `RQL_TYPE_TRANSLATIONS` as a whole